### ISSUE TITLE
rephrase/provide description of Podman

### ISSUE THREAD

**Issue by vrothberg** [containers/libpod#1636]
The [main manpage of Podman](https://github.com/containers/libpod/blob/master/docs/podman.1.md) describes Podman as a debugging tool but over the year it has evolved into a powerful container engine. An openSUSE user asked to rephrase the description of the package but I think we should start here. I have been checking on [podman.io](http://podman.io) and think we could add some "About" page and a short description at the top of the index page as well.

**Comment by vrothberg** [comment]
@TomSweeneyRedHat WDYT?

**Comment by rhatdan** [comment]
SGTM

**Comment by TomSweeneyRedHat** [comment]
@vrothberg In general this sounds good to me.  I'm perfectly fine with the Podman nan page changes and I'm assuming the podman README.md being changed.  Long-term I'm fine with the "about" and the description on podman.io too, but as we talked about in regards to adding a "Documentation" button, I'm just a bit concerned about overloading the buttons on the left side of that page due to scrolling issues that we're trying to solve.   In addition @haircommander just got his head above water with his studies this fall and is talking about a redesign of that page.

So long story short, I'm open to changes to podman.io, please send along any thoughts.  Best bet is to open an issue and/or a PR there.  We'll see if it works for now, but if not we might have to add it later.  But it would be good to at least capture the thinking.

**Comment by rhatdan** [comment]
@vrothberg @TomSweeneyRedHat Did you guys ever change anything?  Can we close this issue?

**Comment by vrothberg** [comment]
@rhatdan, no, I don't think so. What we need is a new description of Podman (i.e., a text in various lengths) that we can then spread over documents, manpages, and the website. When browsing on https://podman.io/ it's still hard to figure out what it actually is. Same applies to Buildah.

**Comment by TomSweeneyRedHat** [comment]
@rhatdan, this bubbled too far down my stack, need to bubble it back up.

**Comment by baude** [comment]
can we close this?
